README.md

```diff
@@ -6,9 +6,13 @@
 The simple package to stub HTTP resource using [WireMock admin](http://wiremock.org/docs/api/)
 
 ## Documentation
+### NPM Documentation
```

Comment:
It is not NPM

Reply:
@oleg-nenashev You're absolutely correct, apologies. Would `Package Documentation` be the correct vernacular?

Reply:
Personally I see no need in section headers that only store a single line